### PR TITLE
Retire Windows services content

### LINKED ISSUE #8417: Out of date
This documentation refers to features that were removed from Visual Studio years ago...

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 02bb2607-4589-179e-55c1-0749dd134ed5
* Version Independent ID: a8dc7f55-6bff-cf9e-9fa5-8c74df09f3b6
* Content: [Walkthrough: Using Client Application Services](https://docs.microsoft.com/en-us/dotnet/framework/common-client-technologies/walkthrough-using-client-application-services)
* Content Source: [docs/framework/common-client-technologies/walkthrough-using-client-application-services.md](https://github.com/dotnet/docs/blob/master/docs/framework/common-client-technologies/walkthrough-using-client-application-services.md)
* Product: **dotnet-framework**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **dotnetcontent**

### PR DESCRIPTION
**Pull request by mairaw**
Fixes #8417

### CODE REVIEW

File: docs/framework/common-client-technologies/index.md [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
+title: "Common client technologies - .NET Framework"
 ms.date: "03/30/2017"
 ms.assetid: 733fc580-337e-4b7c-9756-b70798c18bc3
 author: "mairaw"
 ms.author: "mairaw"
 ---
-# Common Client Technologies in the .NET Framework
+# Common client technologies in the .NET Framework
+
 This section describes different technologies that you can use in your client applications.
```
Comment (rpetrusha) [inline docs/framework/common-client-technologies/index.md]: nit: it now only describes a single technology. I wonder if this portal page to a single technology is even useful.

**Review by rpetrusha — APPROVED** [review]
LGTM, @mairaw. I left a nit -- the Common client technologies topic now is a portal to just a single technology.

File: docs/framework/common-client-technologies/index.md (hunk 1 above)
Comment (mairaw) [reply docs/framework/common-client-technologies/index.md]: I was even thinking if the other technology is even still useful. Maybe I should retire that whole section?

Comment (rpetrusha) [reply docs/framework/common-client-technologies/index.md]: I would tend to agree -- the entire section could be archived.

**Pull request merged**